src/main/java/com/odysseusinc/arachne/datanode/security/AuthenticationTokenFilter.java

```diff
@@ -68,6 +71,11 @@ public void doFilter(
         HttpServletRequest httpRequest = (HttpServletRequest) request;
         String authToken = httpRequest.getHeader(this.tokenHeader);
         try {
+            if (authToken == null && ((HttpServletRequest) request).getCookies() != null) {
```

Comment:
This opens a door for CSRF. Previously we used cookies auth only for selected routes